frontend/src/components/DrawerCollectionList.tsx

```diff
@@ -41,6 +42,18 @@ const DrawerCollectionListItem: React.FC<DrawerCollectionListItemProps> = observ
     return store.drawerSelectedCollection === collectionId
   }
 
+  function hasWhiteSpace(text: string): boolean {
+    return /\s/g.test(text);
+  }
+
+  function wrapTextWithoutSpaces(text: string): string {
+    if (hasWhiteSpace(text)) {
```

Comment:
Maybe we could use ellipsis for any name longer than 30, even with whitespaces and to display full name on the tooltip?
Scenario with 30 letter word without spaces seems very rare, so this condition won't be really used. Maybe we could apply the same logic to collection names? @MaciejWiczk what do you think?

Reply:
You've got a point. The longest I've used is probably DatabaseLibraryUtils, which again, is a library, that could have the same logic applied.

Reply:
@pbylicki @MaciejWiczk 
It should work now well as you mentioned above. Not well solved, but done :D

Reply:
@MaciejWiczk @MarcinMaciaszek So the idea is to have tooltip for all keywords/collections or only to those that are longer than 30 and have ellipsis? For now it is ok, but if we want the second option I could adjust it while working on #130 which affects the same part of the code

Reply:
@pbylicki @MaciejWiczk 
Finally fixed all issues :P sorry have no time
It should work well now, so you can decide if we should merge it.

Reply:
Thanks, good job. Merging it right now.